app/routes.py

```diff
@@ -0,0 +1,291 @@
+def handle_customer(customer_id):
+    if not customer_id.isnumeric():
+        return { "Error": f"{customer_id} must be numeric."}, 400
+    customer_id = int(customer_id)
+    customer = Customer.query.get(customer_id)
+    if not customer:
+        return ({"message": f"Customer {customer_id} was not found"}),404
+
+    if request.method == "GET":
+        return (customer.to_dict()),200
+
+
+    elif request.method == "PUT":
+        form_data = request.get_json()
+        if "name" not in form_data or "postal_code" not in form_data or "phone" not in form_data:
+            return ({
+                "details": "Invalid data"
+            }),400
+        customer.name = form_data['name']
+        customer.postal_code = form_data['postal_code']
+        customer.phone = form_data['phone']
+
+        db.session.commit()
+
+        return ({
+            "name": f"{customer.name}",
+            "phone": f"{customer.phone}",
+            "postal_code": f"{customer.postal_code}"
+        }),200
+
+    elif request.method == "DELETE":
+        db.session.delete(customer)
+        db.session.commit()
+
+        return ({
+            "id": customer.customer_id,
+            "details": f"Customer {customer.name} successfully deleted"
+        }), 200
+
+#RENTAL ROUTES
+
+@rental_bp.route("/check-out", methods = ["POST"])
+def handle_checkout():
+    if request.method == "POST":
+        request_body = request.get_json()
+        if "customer_id" not in request_body:
+            return ({
+                "details": "Request body must include customer_id."
+            }),400
+        elif "video_id" not in request_body:
+            return ({
+                "details": "Request body must include video_id."
+            }),400
+        new_rental = Rental(
+            customer_id = request_body["customer_id"],
+            video_id = request_body["video_id"]
+        )
+        target_video = Video.query.get(new_rental.video_id)
+        if not target_video:
+            return "Video_id doesn’t exist",404
+        target_customer = Customer.query.get(new_rental.customer_id)
+        if not target_customer:
+            return "Customer_id doesn’t exist",404
+        total_inventory = Video.query.get(new_rental.video_id).total_inventory
+        existing_rentals = rentals_associated(new_rental)
+        available_inventory = total_inventory - existing_rentals
+        
+        if available_inventory == 0:
+            return ({"message": "Could not perform checkout"}),400
+
+        db.session.add(new_rental)
+        db.session.commit()
+        return ({
+            "customer_id": new_rental.customer_id,
+            "video_id": new_rental.video_id,
+            "due_date": datetime.datetime.today() - datetime.timedelta(days=7),
+            "videos_checked_out_count": videos_checked_out_count(new_rental),
+            "available_inventory": available_inventory - rentals_associated(new_rental)
```

Comment:
1. I like the calculations done here in the dict.
2. This would be great as a `Rental` helper method (maybe `to_dict`).